android/app/src/main/java/org/fossasia/openevent/utils/SortOrder.java

```diff
@@ -48,7 +48,7 @@ public static String sortOrderSchedule(Context context) {
                 return DbContract.Sessions.START_TIME;
             case SORT_TYPE_SECOND:
                 //By TRACKS
-                return DbContract.Sessions.TITLE;
+                return DbContract.Sessions.TRACK;
```

Comment:
The options says 
`Sort by` `Tracks` or `Start Time`

But when we select Tracks the sessions are sorted alphabetically.
Why would we need sessions alphabetically.
Like the option says we should sort them according to tracks.